### PR TITLE
AB - correct mildot reticle LRPS/AMS/KHS

### PR DESCRIPTION
**Pull request by J3aLeiNe**
correct mildot reticle LRPS/AMS/KHS

### CODE REVIEW

**Comment by J3aLeiNe** [comment]
LRPS/AMS/KHS have now a correct mildot reticle for each zoom, tested with the "User Texture (1m)" and "User Texture (10m)" at respectively 250m and 1000m.

File: addons/scopes/CfgWeapons.hpp [hunk 1]
```diff
@@ -3,89 +3,105 @@ class CfgWeapons {
...
-        ACE_ScopeAdjust_Horizontal[] = { -6, 6 };
+        ACE_ScopeAdjust_Vertical[] = {-4, 30};
+        ACE_ScopeAdjust_Horizontal[] = {-6, 6};
         ACE_ScopeAdjust_VerticalIncrement = 0.1;
         ACE_ScopeAdjust_HorizontalIncrement = 0.1;
         class ItemInfo : InventoryOpticsItem_Base_F {
             class OpticsModes {
                 class Snip {
-                    discreteDistance[] = { 100 };
+					opticsZoomMin = 0.011;
```
Comment (commy2) [inline addons/scopes/CfgWeapons.hpp]: tabs

File: addons/scopes/CfgWeapons.hpp [hunk 2]
```diff
@@ -3,89 +3,105 @@ class CfgWeapons {
...
-        ACE_ScopeAdjust_Horizontal[] = { -6, 6 };
+        ACE_ScopeAdjust_Vertical[] = {-4, 30};
+        ACE_ScopeAdjust_Horizontal[] = {-6, 6};
         ACE_ScopeAdjust_VerticalIncrement = 0.1;
         ACE_ScopeAdjust_HorizontalIncrement = 0.1;
         class ItemInfo : InventoryOpticsItem_Base_F {
             class OpticsModes {
                 class AMS {
-                    discreteDistance[] = { 100 };
+					opticsZoomMin = 0.0285;
```
Comment (commy2) [inline addons/scopes/CfgWeapons.hpp]: tabs

File: addons/scopes/CfgWeapons.hpp [hunk 3]
```diff
@@ -3,89 +3,105 @@ class CfgWeapons {
...
-        ACE_ScopeAdjust_Horizontal[] = { -6, 6 };
+        ACE_ScopeAdjust_Vertical[] = {-4, 30};
+        ACE_ScopeAdjust_Horizontal[] = {-6, 6};
         ACE_ScopeAdjust_VerticalIncrement = 0.1;
         ACE_ScopeAdjust_HorizontalIncrement = 0.1;
         class ItemInfo : InventoryOpticsItem_Base_F {
             class OpticsModes {
                 class KHS {
-                    discreteDistance[] = { 100 };
+					opticsZoomMin = 0.026;
```
Comment (commy2) [inline addons/scopes/CfgWeapons.hpp]: tabs

File: addons/scopes/CfgWeapons.hpp [hunk 4]
```diff
@@ -3,89 +3,105 @@ class CfgWeapons {
...
```
Comment (commy2) [inline addons/scopes/CfgWeapons.hpp]: tabs

**Comment by commy2** [comment]
PR still is using tabs

**Comment by Glowbal** [comment]
Thanks @J3aLeiNe

**Pull request merged**